star_tides/services/databases/mongo/schemas/user_schema.py

```diff
@@ -0,0 +1,10 @@
+''' star_tides.services.databases.mongo.schemas.user_schema
+'''
+import marshmallow_mongoengine as ma
+from star_tides.services.databases.mongo.models.user_model import UserModel
+
+
+class UserSchema(ma.ModelSchema):
+    class Meta:
+        model = UserModel
+        exclude = ['password', 'kb_privilege']
```

Comment:
You can see the justification in the Design Decisions section of the wiki

Reply:
Thanks for writing that up. Might be good to add/link code examples. Can you also enforce writes to the DB using *Model instead of *Schema (or vice versa) using types?

Reply:
Wiki stuff looks great in general, especially the page about controllers and actions. Thanks!

Reply:
What object gets written to the DB? The UserSchema or the UserModel? Is the *Schema just a way to structure DB docs? I would just like to avoid a scenario with (due to loose typing) we have `db.write(user)` and it's a non-Model (or w/e) object and we write junk data.

Reply:
So there is no `db.write` method. You have to create/update/delete a model object to interact with the database. In the definition of the model class [here](https://github.com/STAR-TIDES/kb/blob/main/star_tides/services/mongo/models/user_model.py#L6) it inherits from a class called `Document`. The methods inherited from `Document` handle all of the database interactions for the instantiated object.